rules.txt

```diff
@@ -228,6 +228,7 @@
 /posts/office-of-sustainability/2017-03-28-environment-sustainability-city-of-philadelphia-action-guide 301 /2017-11-29-environmental-protection-during-the-trump-administration/
 /potholes                                 301  http://www.philadelphiastreets.com
 /ppr                                      301  /departments/philadelphia-parks-recreation/
+/ParksandRecreation/(.*)                  301  /departments/philadelphia-parks-recreation/ 

```

Comment:
oh, also this can be lowercase because the regex rules are made case insensitive. NBD though either way. good to go!